### PR TITLE
Add exercise 4

### PR DESCRIPTION
**Pull request by nguyenvanman**

### CODE REVIEW

File: Exercise__1/Controller/OpenweathermapServices.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+using Android.Views;
+using Android.Widget;
+
+namespace Exercise__1.Controller
+{
+    public class OpenweathermapServices
+    {
+        public string RootUrl = "http://api.openweathermap.org";
+
+        public
```
Comment (voquanghoa) [inline Exercise__1/Controller/OpenweathermapServices.cs]: Wtf

File: Exercise_4/Controllers/OpenWeatherMapServices.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+        private readonly RestClient restClient = new RestClient(RootUrl);
+
+        private const string ApiKey = "b2baa28658ef1bcc7bb06834dce9b058";
+
+        public WeatherInfo GetWeatherByCityName(string cityName)
+        {
+            var request = new RestRequest($"/data/2.5/weather?q={cityName}&appid={ApiKey}", Method.GET);
+            var response = restClient.Execute(request);
+            return (response.StatusCode.ToString().Equals("OK"))
```
Comment (voquanghoa) [inline Exercise_4/Controllers/OpenWeatherMapServices.cs]:
```C#
if(response.StatusCode == HttpStatusCode.OK)...
```

File: Exercise_4/MainActivity.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            Init();
+        }
+
+        private void Init()
+        {
+            svCityName.QueryTextSubmit += (s, e) =>
+            {
+                var service = new OpenWeatherMapServices();
+                var weatherInfo = service.GetWeatherByCityName(svCityName.Query);
+                if (weatherInfo != null)
```
Comment (voquanghoa) [inline Exercise_4/MainActivity.cs]: Handle exception instead

**Comment by voquanghoa** [comment]
Please fix conflicts

File: Exercise_4/Controllers/OpenWeatherMapServices.cs (hunk 2 above)
Comment (nguyenvanman) [reply Exercise_4/Controllers/OpenWeatherMapServices.cs]: Done

File: Exercise_4/MainActivity.cs (hunk 3 above)
Comment (nguyenvanman) [reply Exercise_4/MainActivity.cs]: Done

File: Exercise__1/Controller/OpenweathermapServices.cs (hunk 1 above)
Comment (nguyenvanman) [reply Exercise__1/Controller/OpenweathermapServices.cs]: Deleted

**Pull request merged**